aptos-move/move-examples/messageboard/sources/ACL.move

```diff
@@ -0,0 +1,46 @@
+/// Access control list (ACL) module. An ACL is a list of account addresses who
```

Comment:
same, as offer, if we're going to recommend using this, we should just make it generally available. That would also reduce burden on the chain, no?

Reply:
added to stdlib